<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Worker</class>
 <widget class="QDialog" name="Worker">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>761</width>
    <height>492</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>282</width>
    <height>212</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>10000</width>
    <height>14342432</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_6">
   <item row="0" column="0" rowspan="2">
    <widget class="QGroupBox" name="countWorker_2">
     <property name="maximumSize">
      <size>
       <width>465546</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Количество сотрудников</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Введите кол-во: </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="worker"/>
          </item>
          <item>
           <widget class="QPushButton" name="countWorker">
            <property name="text">
             <string>Задать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <widget class="QTableWidget" name="tableWidget"/>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_2">
          <item>
           <spacer name="horizontalSpacer">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="close_Btn">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="saveWorker">
            <property name="text">
             <string>ОК</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QGroupBox" name="Choice">
     <property name="maximumSize">
      <size>
       <width>178957</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Добавление</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_3">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_6">
          <property name="text">
           <string>                        Добавить 1-го сотрудника или несколько?</string>
          </property>
         </widget>
        </item>
        <item>
         <layout class="QVBoxLayout" name="verticalLayout_3">
          <item>
           <widget class="QCheckBox" name="OneWorker">
            <property name="text">
             <string>Одного</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QCheckBox" name="Workers">
            <property name="text">
             <string>Несколько</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <spacer name="horizontalSpacer_3">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="cancel">
            <property name="text">
             <string>Отмена</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="OK">
            <property name="text">
             <string>Выбрать</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="Choice_DeleteWorker">
     <property name="title">
      <string>Увольнение сотрудника</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_4">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QCheckBox" name="All_deleteWorker_check">
          <property name="text">
           <string> уволить всех найденных с таким номером сотрудников</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QCheckBox" name="delete_WorkerChoice_check">
          <property name="text">
           <string>уволить сотрудника по выбору</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_9">
        <item>
         <spacer name="horizontalSpacer_4">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="back_find_worker_phone_delete">
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="delete_phone_Btn">
          <property name="text">
           <string>OK</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="0">
    <widget class="QGroupBox" name="new_Worker">
     <property name="maximumSize">
      <size>
       <width>6436543</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="title">
      <string>Новый сотрудник</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_2">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="surname_label">
            <property name="text">
             <string>Фамилия:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="surname"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="name_label">
            <property name="text">
             <string>    Имя:    </string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="name"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="patronymic_label">
            <property name="text">
             <string> Отчество:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="patronymic">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="phone_label">
            <property name="text">
             <string>Номер телефона:</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="countPhone_label">
            <property name="text">
             <string/>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLineEdit" name="phone"/>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <spacer name="horizontalSpacer_2">
            <property name="orientation">
             <enum>Qt::Horizontal</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>40</width>
              <height>20</height>
             </size>
            </property>
           </spacer>
          </item>
          <item>
           <widget class="QPushButton" name="Back_ChoiceAddWorker_Btn">
            <property name="text">
             <string>Назад</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="newWork_btn">
            <property name="text">
             <string>ОК</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QGroupBox" name="deleteChoice_Box">
     <property name="title">
      <string>Уваолить сотрудника</string>
     </property>
     <layout class="QGridLayout" name="gridLayout_5">
      <item row="0" column="0">
       <layout class="QVBoxLayout" name="verticalLayout_6">
        <item>
         <widget class="QTextBrowser" name="Worker_textBrowser"/>
        </item>
        <item>
         <widget class="QLabel" name="deleteWorkerChoice_label">
          <property name="text">
           <string>Выберите номер сотрудника которого хотите удалить</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLineEdit" name="deleteWorker_line"/>
        </item>
       </layout>
      </item>
      <item row="1" column="0">
       <layout class="QHBoxLayout" name="horizontalLayout_10">
        <item>
         <spacer name="horizontalSpacer_5">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
        <item>
         <widget class="QPushButton" name="back_Choice_DeleteWorker_Btn">
          <property name="text">
           <string>Назад</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="ChoiceDelete_Btn">
          <property name="text">
           <string>Уволить</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
